expression/aggregation.go

```diff
@@ -280,6 +452,41 @@ func (cf *concatFunction) Update(row []types.Datum, groupKey []byte, ectx contex
 	return nil
 }
 
+func (cf *concatFunction) StreamUpdate(row []types.Datum, ectx context.Context) error {
+	ctx := cf.getStreamedContext()
+	vals := make([]interface{}, 0, len(cf.Args))
```

Comment:
Why not define `vals` as `[]types.Datum`, and use `ToString`?
I'm afraid for some types like `mysql.Time` or `mysql.MyDeciaml`, the result of `fmt.Sprintf("%v", val` would be incorrect.
Probably this is not covered in test.